embulk-filter-rename/src/main/java/org/embulk/filter/rename/RenameFilterPlugin.java

```diff
@@ -441,12 +469,27 @@ private Schema applyUniqueNumberSuffixRule(Schema inputSchema, UniqueNumberSuffi
         return outputBuilder.build();
     }
 
-    private static final ImmutableMap<String, String> CHARACTER_TYPE_KEYWORDS =
-            new ImmutableMap.Builder<String, String>()
-                    .put("a-z", "a-z")
-                    .put("A-Z", "A-Z")
-                    .put("0-9", "0-9")
-                    .build();
+    private static String repeatStrings(final String string, final int count) {
+        final StringBuffer buffer = new StringBuffer();
```

Comment:
Nit: StringBuilder would suffice?

Reply:
Umm, I don't remember why I used `StringBuffer`, but `StringBuilder` could be better. I'll have a revisit here later. Thanks!